app/partials/disease-associations.html

```diff
@@ -98,11 +101,11 @@ <h3 view-title>{{search.total}} {{search.total===1 ? "target" : "targets"}} asso
                                     <div ng-if="fuzzyTargetArray.length > 0">
                                         <div >
                                             <div  class="checkbox cttv-facet-checkbox clearfix"
-                                                    title="These are the target names from the file for which exact match was not found but fuzzy match got similar enough name. By default they are included in association search. Clicking on the checkbox will exclude them from the search. ">
+                                                    title="These are the target names from the file for which exact match was not found but similar name was found. By default they are included in association search. Clicking on the checkbox will exclude suggested target names from the search. ">
```

Comment:
I find this sentence too long. Alternative:
"""
Targets from the file without an exact match for which a similar term was found. Tick or untick the checkbox to include or exclude them from the association table
"""